src/services/startupChecks.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+
+const red = "\x1b[31m";
+const bold = "\x1b[1m";
+const reset = "\x1b[0m";
+
+function assertEnvVarsPresent() {
+  let variables = [
+    'DATABASE_URI',
+    'UPLOADPATH'
+  ];
+
+  // color codes for colored output in terminals
+  let canStart = true;
```

Comment:
With the name canStart, it feels like the boolean values should be flipped. Checking that canStart is true and then showing an error that you can't start because of missing variables is a bit confusing.